**Задача 1.**
*Известно, что генеральная совокупность распределена нормально, со средним квадратическим отклонением, $\displaystyle\sigma=16$. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95, если выборочная средняя M=80, а объем выборки n=256*

Поскольку нам известно среднее квадратическое отклонение генеральной совокупности $\sigma=16$, то для расчета $95\%$-го доверительного интервала, будем использовать Z-критерий. Табличное значение Z-критерия, и для $95\%$-го доверительного интервала $z_{\alpha/2}=1,96$   (много раз уже смотрели на лекции))

95\%-й доверительный интервал расчитаем как: 

In [1]:
left=80-1.96*16/256**(1/2)
right=80+1.96*16/256**(1/2)
print(f'[{left};{right}]')

[78.04;81.96]


**Ответ:**

Доверительный интервал для оценки математического ожидания с надежностью 0.95 равен   $\left[78.04;81.96\right]$  

---

**Задача 2.**
*В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.*

In [2]:
import numpy as np
import scipy.stats as stats

Поскольку из условия задачи нам ничего не известно ни о мат. ожидании генеральной совокупности ни о среднем квадратическом отклонении для генеральной совокупности, то для расчета $95\%$-го доверительного интервала будем использовать $t$-критерий,  и формулу:  
$\displaystyle\bar X\pm t_{\alpha/2}\cdot\frac{\sigma_{несмещ}}{\sqrt{n}}$

In [3]:
arr=np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])
print(f'Среднее выборочное: {np.mean(arr): .2f},\n'
      f'Размер выборки n={len(arr)},\n'
      f'Среднее квадратическое отклонение по выборке(несмещенное): {np.std(arr, ddof=1): .2f}.'
     )

Среднее выборочное:  6.59,
Размер выборки n=10,
Среднее квадратическое отклонение по выборке(несмещенное):  0.45.


In [4]:
def t_from_table(confidens, len_array):
    alpha=(1-confidens)
    return stats.t.ppf(1-alpha/2, len_array-1)
print(f'Таб.значение t-критерия для 95%-го доверительного интервала выборки: {t_from_table(0.95, len(arr)): .3f}')

Таб.значение t-критерия для 95%-го доверительного интервала выборки:  2.262


In [5]:
def confidens_int(arr, confidens):
    return round(np.mean(arr)-t_from_table(confidens,len(arr))*np.std(arr, ddof=1)/len(arr)**0.5,3), \
           round(np.mean(arr)+t_from_table(confidens,len(arr))*np.std(arr, ddof=1)/len(arr)**0.5,3)

confidens_int(arr, 0.95)

(6.268, 6.912)

**Ответ:**

Доверительный интервал для истинного значения Х с доверительной вероятностью 0,95 равен $\left[6.268; 6.912\right]$  

---

**Задача 3.**
*Рост дочерей 175, 167, 154, 174, 178, 148, 160, 167, 169, 170 Рост матерей 178, 165, 165, 173, 168, 155, 160, 164, 178, 175
Используя эти данные построить 95% доверительный интервал для разности среднего роста родителей и детей.*

In [16]:
daughters = np.array([175, 167, 154, 174, 178, 148, 160, 167, 169, 170])
mothers = np.array([178, 165, 165, 173, 168, 155, 160, 164, 178, 175])

mean_diff = np.mean(mothers) - np.mean(daughters)
std = np.sqrt((np.std(mothers) ** 2) / len(mothers) + (np.std(daughters) ** 2) / len(daughters))

d1 = np.var(mothers, ddof=1) 
d2 = np.var(daughters, ddof=1) 

d = (d1 + d2)/2 #
SE= np.sqrt (d/10+d/10) 

t = stats.t.ppf(0.975, 18) 


c_interval = (mean_diff -  t * SE, mean_diff +  t * SE)
c_interval

(-6.268418034506846, 10.068418034506857)

**Ответ**

95% доверительный интервал разности среднего роста родителей и детей: $\left[-6.268; 10.068\right]$ 